src/index.ts

```diff
@@ -12,33 +13,67 @@ import { looper } from './utils/looper'
 import { parseConfig } from './utils/parse-config'
 import { validateConfig } from './utils/validate-config'
 
+import {
+  closeLog,
+  openLogfile,
+  printAllLogs,
+  flushAllLogs,
+} from './utils/history'
+
 class Monika extends Command {
-  static description = 'describe the command here'
+  static description = 'Monika command line monitoring tool'
 
   static flags = {
     // add --version flag to show CLI version
     version: flags.version({ char: 'v' }),
     help: flags.help({ char: 'h' }),
-    // flag with a value (-n, --name=VALUE)
+
     config: flags.string({
       char: 'c',
       description:
-        "JSON configuration file path e.g './config.json' (default './config.json')",
-      required: true,
+        'JSON configuration filename. If none is supplied, will look for default in current path.',
```

Comment:
I think we should add information about the "default file" name. Probably add (Default: './config.json') or similar?

Reply:
So that user knows what file would be read as a default file

Reply:
![image](https://user-images.githubusercontent.com/964106/111724024-15ac8880-8897-11eb-921d-20c07ad06184.png)

yup agree.